dist/common/scripts/scylla_io_setup

```diff
@@ -102,6 +103,15 @@ class scylla_cpuinfo:
         else:
             return len(self._cpu_data["system"])
 
+def configure_iotune_open_fd_limit(shards_count):
+    required_fds_count = 10 * shards_count
+    soft_limit, hard_limit = resource.getrlimit(resource.RLIMIT_NOFILE)
+
+    if soft_limit < required_fds_count or hard_limit < required_fds_count:
+        soft_limit = max(soft_limit, required_fds_count)
+        hard_limit = max(hard_limit, required_fds_count)
+        resource.setrlimit(resource.RLIMIT_NOFILE, (soft_limit, hard_limit))
```

Comment:
nit: do we wish to catch and error here if setrlimit raises it?

Reply:
Sure, fixed :+1: